frontend/components/create-review/review-form.tsx

```diff
@@ -479,10 +485,16 @@ function ReviewForm(): JSX.Element {
 							{t('create-review.review-form.reset')}
 						</ButtonLight>
 						<Button
-							disabled={!token || !disclaimer || maliciousStringDetected}
```

Comment:
Added a loading spinner since it can take up to 5sec to finish the review filtering.

Running into a weird error here, says I can't put an element in here, will let the tailwind expert sort this one out ;)